completions/ip

```diff
@@ -318,6 +318,24 @@ _ip()
             esac
             ;;
 
+        netns)
+            case $subcmd in
+                add|identify|list|list-id|monitor)
```

Comment:
list, list-id, and monitor don't seem to be actual TODOs, because they take no args. So for clarity, I think they should be in a separate case.

Reply:
list-id seems to take optional arguments, so it should remain in TODO. I'll split off the other two.